renderdoc/api/replay/renderdoc_replay.h

```diff
@@ -415,6 +430,25 @@ inline const WindowingData CreateAndroidWindowingData(ANativeWindow *window)
   return ret;
 }
 
+DOCUMENT(R"(Create a :class:`WindowingData` for a Wayland ``wl_surface`` handle.
+
+:param struct wl_display *display: The native ``wl_display`` connection used for this window.
+:param struct wl_surface *window: The native ``wl_surface`` handle for this window.
```

Comment:
These comments are formatted for the python bindings first (with being good interface docs for C++ a close second), so the pointer syntax won't necessarily show up well. It's still clear enough if you just put `:param wl_display display` and that will definitely show correctly.

Reply:
Ah, I didn't know that. Sure, I'll remove the `struct` keyword and pointer syntax.